packages/kit/src/runtime/load.js

```diff
@@ -4,10 +4,19 @@
  */
 export function normalize(loaded) {
 	// TODO should this behaviour be dev-only?
+	const hasStatusError =
```

Comment:
This should be called `has_error_status` because we use underscores for internal variables. We only use camel case for public APIs